frpc.toml

```diff
@@ -1,10 +1,10 @@
-serverAddr = "52.77.216.244"
+serverAddr = "gaianet.network"
 serverPort = 7000
 metadatas.deviceId = ""
 
 [[proxies]]
-name = "demo.gaianet.xyz"
+name = "{your_subdomain}.gaianet.network"
```

Comment:
This is a little confusing since it implies that `us.gaianet.network` might not work (although it did work). Maybe change to `${your_subdomain}.${serverAddr}` or something like that?